api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -74,8 +74,7 @@
 
   private Function<Timestamp, Function<DbUser, DbUser>> registerUserWithTime =
       t -> dbu -> registerUser(t, dbu);
-  private Function<DbUser, DbUser> registerUserNow =
-      registerUserWithTime.apply(new Timestamp(PROVIDED_CLOCK.millis()));
```

Comment:
this was sometimes receiving clock values from other tests, so I moved it after we defined the clock